lib/better_errors/error_page.rb

```diff
@@ -112,7 +113,11 @@ def inspect_value(obj)
     end
 
     def inspect_raw_value(obj)
-      value = CGI.escapeHTML(obj.inspect)
+      if defined?(ObjectSpace.memsize_of) 
+        value =  CGI.escapeHTML(ObjectSpace.memspace_of(obj)) 
```

Comment:
The `value` is the HTML representation of the variable. So setting `value` here to the size of the variable is not what you want to do.

If we can decide whether it's small enough to inspect using `ObjectSpace`, we want to inspect the variable as HTML, and return that. If we can't use `ObjectSpace.memsize_of`, then we need to inspect it anyway and then check the size of the rendered HTML. So this method will need to be rewritten.